server/src/server.ts

```diff
@@ -155,34 +160,56 @@ parser.setIncludeFileFetch(async (stringUri: string, includeString: string) => {
 					path: cleanString
 				}).toString();
 			}
+			validUri = await validateUri(cleanString, currentUri.scheme);
 			break;
 
 		case `file`:
+			const workspaceFolders = await connection.workspace.getWorkspaceFolders();
+			let workspaceFolder: WorkspaceFolder | undefined;
+			if (workspaceFolders) {
+				workspaceFolder = workspaceFolders.find(folderUri => uriPath.startsWith(URI.parse(folderUri.uri).path))
+			}
+
 			cleanString = includeString;
-			
+
 			if (cleanString.startsWith(`'`) && cleanString.endsWith(`'`)) {
 				cleanString = cleanString.substring(1, cleanString.length - 1);
 			}
+
+			if (Project.isEnabled) {
```

Comment:
If project mode is enabled, we will do a local search as we were doing before.

If it is disabled, the search is disabled and the path is searched explicitly.